checker/jtreg/Lintoption/InstanceofLintOptionEnabled.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * @test
+ * Test case for instanceof lint option: -Alint=instanceof
+ * @requires jdk.version >= 17
+ * @compile/ref=InstanceofLintOptionEnabled.out -XDrawDiagnostics -processor org.checkerframework.checker.tainting.TaintingChecker InstanceofLintOptionEnabled.java -Alint=instanceof
+ */
```

Comment:
```suggestion
 * @compile/ref=InstanceofLintOptionEnabled.out -XDrawDiagnostics -processor org.checkerframework.checker.tainting.TaintingChecker InstanceofLintOptionEnabled.java
 * @compile/ref=InstanceofLintOptionEnabled.out -XDrawDiagnostics -processor org.checkerframework.checker.tainting.TaintingChecker InstanceofLintOptionEnabled.java -Alint=instanceof
 * @compile/ref=InstanceofLintOptionEnabled.out -XDrawDiagnostics -processor org.checkerframework.checker.tainting.TaintingChecker InstanceofLintOptionEnabled.java -Alint=instanceof:unsafe
 */
```

To test that the warning is issued by default and with the more detailed lint key.